mysite/settings.py

```diff
@@ -79,9 +79,9 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'codingclubiitg',
-        'USER': 'coding-club',
-        'PASSWORD': 'coding-club',
+        'NAME': 'postgres',
+        'USER': 'postgres',
+        'PASSWORD': 'tbasic123RM',
         'HOST': 'localhost',

```

Comment:
Okay, I made a mistake here. Because the postgresql wasn't accepting the default credentials and I wasn't able to run the server on my laptop, i made this changes. Have to revert it

Reply:
I restored the settings.py database part with the default one now. Only Event model has been modified now in models.py to add the string representation

Reply:
Hey ripun, make another pull request on branch 'latest' with your work. The master branch has unnecessary deployment changes.